src/reducers/ByProgramReducer.js

```diff
@@ -10,7 +10,7 @@ export const initialState = ({ transactionType }) => ({
   period: null,
   orderType: null,
   name: '',
-  currentKey: 'program',
```

Comment:
Kind of slipped in, was checking if this needed an initial value. It doesn't. Can revert if needed.